### ISSUE TITLE
Request for a tutorial about "make feature" model

### ISSUE THREAD

**Issue by zhengjiasen** [hackalog/bus_number#73]
I learned the "notebook/*.ipynb" tutorial many times, I love it, but the tuorial has not about the "make feature" step, could you give an application of feature making, such as filling missing value, new feature construction in training set and test set

**Comment by zhengjiasen** [comment]
no answer
